CMakeLists.txt

```diff
@@ -215,6 +215,10 @@ option(SWIFT_STDLIB_ENABLE_RESILIENCE
     "Build the standard libraries and overlays with resilience enabled; see docs/LibraryEvolution.rst"
     FALSE)
 
+option(SWIFT_STDLIB_ENABLE_REFLECTION_METADATA
+    "Build the standard libraries and overlays with remote reflection metadata; see docs/proposals/RemoteMirrors.rst"
+    FALSE)
+
```

Comment:
@bitjammer Well, at the very least we have to shake out all the bugs turning this on might expose. When we turn it on for real, it should be changed in the frontend, not in CMake.